pepper-apis/src/main/java/org/broadinstitute/ddp/transformers/InstantStringAdapter.java

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.ddp.transformers;
+
+import java.io.IOException;
+import java.time.Instant;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonToken;
+import com.google.gson.stream.JsonWriter;
+
+/**
+ * Adapter to convert an instant to/from a string. This is null-safe so no need to call `nullSafe()` wrapper.
+ */
+public class InstantStringAdapter extends TypeAdapter<Instant> {
+
+    @Override
+    public Instant read(JsonReader jsonReader) throws IOException {
+        if (jsonReader.peek() == JsonToken.NULL) {
+            jsonReader.nextNull();
+            return null;
+        } else {
+            return Instant.parse(jsonReader.nextString());
+        }
+    }
+
+    @Override
+    public void write(JsonWriter jsonWriter, Instant instant) throws IOException {
+        if (instant == null) {
+            jsonWriter.nullValue();
+        } else {
+            jsonWriter.value(instant.toString());
```

Comment:
Could we rename to something like `InstantToIsoDateAdapter` to make intent more explicit?

Reply:
The format seems to be called "iso instant format"... I guess I'll call it `InstantToIsoInstant`?

Reply:
In my mind `Instant` is just a Java thing. I think idea is to transform to a consumable JSON format. The format we are targeting is ISO-8601, so the IsoDateAdapter suffix seemed more descriptive to me. Also aligns with the some of the other Adapters that have similar mission. But this is an opinion thing. Use your best judgement.

Reply:
For me, "iso date" isn't quite right. There is both a "date" and a "time" part. Not exactly sure what ISO-8601 calls it, maybe just "DateTime". I think "iso instant" captures the idea as well, so I'll stick with that for now.

Reply:
Java JDK calls it ISO Date Time https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html#ISO_DATE_TIME

Reply:
Unfortunately, the Java `Instant` class won't be able to parse the ISO Date Time format that's permitted in that URL link. It only does the `Z` timezone, as indicated by the related but different [`ISO_INSTANT` formatter](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html#ISO_INSTANT).

Reply:
I agree with @MocanaAtBroad on this- `IsoDateTimeAdaptor` is a name more apt to its use. It's converting a string to and from an ISO-8601 DateTime format. The external representation here is more important than the internal one.